app/models/post.rb

```diff
@@ -1,4 +1,6 @@
 class Post < ApplicationRecord
+  validates :comment_counter, :likes_counter, numericality: { greater_than_or_equal_to: 0 }
+  validates :title, presence: true, length: { maximum: 250 }
   belongs_to :author, class_name: 'User'
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
```

Comment:
Please use flash messages in `comments#create` action for successful and not successful path and render it in views.